src/test/java/org/example/homework13/StringUtilsTests/StringUtilsTests.java

```diff
@@ -0,0 +1,113 @@
+package org.example.homework13.StringUtilsTests;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class StringUtilsTests {
+    @Test
+    @Tag("1st")
+    @DisplayName("Check user's email basic correctness")
+    void checkEmail() {
+        String userEmail = "kateryna.test@gmail.com";
+
+        assertTrue(StringUtils.isNoneEmpty(userEmail));
+        assertTrue(StringUtils.contains(userEmail, '@'));
+        assertTrue(StringUtils.endsWith(userEmail, ".com"));
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"Andy", "Anna", "Albul"})
+    @Tag("2nd")
+    @DisplayName("Check that all user's name start with A")
```

Comment:
talking about parametrized test it is considered as 3 tests with different data entry. so the following naming would be better
```suggestion
    @DisplayName("Check that the name start with A")
```